static/app/bootstrap/initializeSdk.tsx

```diff
@@ -144,6 +150,21 @@ export function initializeSdk(config: Config, {routes}: {routes?: Function} = {}
       if (isFilteredRequestErrorEvent(event) || isEventWithFileUrl(event)) {
         return null;
       }
+
+      event.breadcrumbs = event.breadcrumbs?.filter(crumb => {
```

Comment:
In theory we have a `beforeBreadcrumb` hook that works similar to `beforeSend` but is more isolated to breadcrumbs. It is probably preferrable to this logic. If you wanna drop a breadcrumb you can simply return `null` in the `beforeBreadcrumb` hook.

Reply:
oh nice, even better

Reply:
agree that it is better than handling this in the UI

Reply:
Updated! much cleaner, thanks for jumping in on this @lforst